microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -26,7 +26,7 @@ To add and manage users in the Microsoft Edge program to manage extensions, you
 | _GitHub account_ | A user account at GitHub.com.  You can use your personal GitHub account to sign in to Partner Center—a Microsoft account (MSA) will be created for you. |
 | _Partner Center account_, _Partner Center developer account_ | A _Partner Center account_ is an account on partner.microsoft.com.  To submit Microsoft Edge extensions, you need a _Partner Center developer account_, which is a Partner Center account that has a Microsoft account (MSA) as the Primary Owner. |
 | _Microsoft Edge Program account_ | Enables multiple users to work with Microsoft Edge extensions at Partner Center. |
-| _Microsoft Entra ID_ | A Microsoft Entra ID account. |
+| _Microsoft Entra ID_ | A [Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/whatis) account. |
```

Comment:
A few comments about links:

* We usually don't link mid-sentence words to avoid distracting from reading. Instead of linking the thing that people are reading, we instead add another sentence afterwards, like: "A Microsoft Entra ID. To learn more, see _link_"
* Please always remove the locale part of URLs (/en-us)
* When linking to pages that are on other learn.microsoft.com docsets, please use URLs that start with / instead. So in this case: /entra/fundamentals/whatis.

Reply:
See also the build report: https://github.com/MicrosoftDocs/edge-developer/pull/3370#issuecomment-2642151096

Reply:
```
| _Microsoft Entra ID_ | A Microsoft Entra ID account; see [What is Microsoft Entra ID?](/entra/fundamentals/whatis) |
```